packages/client/src/pages/Home/index.js

```diff
@@ -273,18 +313,32 @@ const HomePage = () => {
               )
             })}
           </Grid><br /><br />
-          <Button theme="accent" size="md">{t('home.MenteesLearnMoreButton')}</Button>
-          <Button theme="accent" size="md">{t('home.MentorsLearnMoreButton')}</Button>
+          <span>{t('home.learnMore')}</span><br /><br />
+          <a href="/parents"><Button theme="accent" size="md">{t('home.MenteesLearnMoreButton')}</Button></a>
+          <a href="/mentors"><Button theme="accent" size="md">{t('home.MentorsLearnMoreButton')}</Button></a>
         </Section>
-        <Section p="100px">
-          statistics here
+        <Section p="75px">
+          <h2>{t('home.hearFromUs')}</h2><br />
+          {/* <p className="title">{t('home.testimonials.testimonial6')}</p>
+          <p>{t('home.testimonialauths.testimonial6auth')}</p> */}
+          <Carousel>
+            {Testimonials.map((s) => 
+              <div key={s.key}>
+                <h3>{t(s.text)}</h3>
+                <p>{t(s.auth)}</p>
+              </div>
+            )}
+          </Carousel>
+          {/* carousel currently kind of ugly and i don't think formatting can be changed */}
+          {/* may have to custom create Carousel in components and dictate formatting there, 
+          but i'm not sure how to actually put all of the pieces together */}
```

Comment:
If you want to use this particular library for the carousel you have to the dig into css to style the arrows. If you look into some of the other project files you will see how we do it using "styled components."
I also found a doc page on how to adjust the style : https://sag1v.github.io/react-elastic-carousel/styling

Alternatively, this carousel looks good to be out of the box although it might be missing that personal touch. But, that can be done at a later time with css too.
https://learus.github.io/react-material-ui-carousel/